src/foam/u2/grid/GridCell_old.js

```diff
@@ -0,0 +1,347 @@
+foam.CLASS
+({
+    package: "foam.u2.grid",
+    name: "GridCell",
+    extends: "foam.u2.Element",
+
+    imports: [
+    ], 
+    
+    axioms: [
+      foam.u2.CSS.create({
+          code: function() {/*
+                ^grid-cell {
+                    border: 1px solid black; 
+                    align: center; 
+                }
+                ^row-highlight {
+                    background-color:rgba(0, 0, 255, 0.1);
+                }
+                ^col-highlight {
+                    background-color:rgba(0, 255, 0, 0.1);
+                }
+                ^intersection-highlight {
+                    background-color:rgba(255, 255, 0, 0.4);
+                }
+                ^selected{
+                   border: 2px solid #f00;
+                    padding: 2px;
+                }
+          */}
+        })
+      ],
+    
+    properties: [
+        'of',
+        "selectedEntry",
+        {
+            name: 'entrySelectable',
+            value: false, 
+        }, 
+        {
+            name: "cell",
+            factory: function(){
+                var b  = foam.u2.Element.create();
+                b.setNodeName("div");
+                return b; 
+            }
+        },
+        {
+            name: "cellBottom",
+            factory: function(){
+                var b  = foam.u2.Element.create();
+                b.setNodeName("div");
+                return b; 
+            }
+        },
+        {
+            class: 'Class', //e.g.WorkorderCellView
+            name: 'cellView',  
+        },
+        {
+            name: 'order', 
+        }, 
+    
+        'colProperty',
+        'rowProperty',
+        'colMatch',
+        'rowMatch',
+        //might not be an id, might be an object, date, etc. 
+        {
+            name: 'colMatchId',
+            expression: function(colMatch, matchColId){
+               if (matchColId && colMatch ){
+                return colMatch.id; 
+               }else {
+                return colMatch; 
+               }
+            }
+        }, 
+        {
+            name: 'rowMatchId',
+            expression: function(rowMatch, matchRowId){
+               if (matchRowId && rowMatch ){
+                return rowMatch.id; 
+               }else {
+                return rowMatch; 
+               }
+            }
+        }, 
+        {
+            name: 'selected',
+            value: false, 
+        },
+        {
+            name: 'matchRowId',
+            value: false, 
+        },
+        {
+            name: 'matchColId',
```

Comment:
It's okay to just reply "fixed" or "done" if you've made the change without any questions.